src/main/resources/db/migration/common/V1_100__temporary_database_tweaks.sql

```diff
@@ -0,0 +1 @@
+ALTER SEQUENCE evh_seq RESTART WITH 400;
```

Comment:
Please see my earlier comment - it was previously 378. Think this should be done in the `V1_101__insert_event_handler_reference_data.sql` script

Reply:
done